<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cube Debug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Device connection</string>
    </property>
    <widget class="QLineEdit" name="lineEditIP">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>106</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDisconnect">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>106</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug level setting</string>
    </property>
    <widget class="QPushButton" name="pushButtonApplyDebug">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxComp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>COMP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CLIA</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxClia">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxSo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRd">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxClient">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>RD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CLIENT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Touch</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxTouch">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxJoystick">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>52</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Joystick</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Log setting</string>
    </property>
    <widget class="QLineEdit" name="lineEditLog">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStartRecord">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Start record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStopRecord">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop record</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1031</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Log preview</string>
    </property>
    <widget class="QTextBrowser" name="textBrowserLogPreview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1011</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCopy">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CopyToClipboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
